tests/knocking_test.go

```diff
@@ -1,7 +1,9 @@
-// +build msc2403
+// Knocking is not yet implemented on Dendrite
+//go:build !dendrite_blacklist
```

Comment:
This line was automatically added by my IDE. I assume it's correct to include?

Reply:
That's fine.